tests/ops/api/v1/endpoints/test_connection_template_endpoints.py

```diff
@@ -1081,7 +1081,7 @@ def test_get_connection_secret_schema_mssql(
                     "type": "string",
                 },
             },
-            "required": ["host", "username", "password", "dbname"],
+            "required": ["host", "username", "password"],
```

Comment:
What does this test do? Just checks that we don't have to provide `dbname` as a default any more? Could we get a super simple test that just passes some two different values for `dbname` to prove we can do the different types?

Reply:
Good enough!